openfisca_france_local/convert_benefit_conditions_to_parameters.py

```diff
@@ -0,0 +1,152 @@
+        parameter_operator: str = comparison_operators[condition['operator']]
+        return {
+            condition_type: {
+                parameter_operator: {
+                    'values': {date: {
+                        'value': condition['value']}},
+                    }
+                }
+            }
+
+    def generate_quotient_familial_parameter(condition: dict) -> dict:
+        parameter_operator: str = comparison_operators[condition['operator']]
+
+        return {
+            condition_type: {
+                condition['period']: {
+                    parameter_operator: {
+                        'values': {date: {
+                            'value': condition['value']
+                            }}
+                        }
+                    }
+                }
+            }
+
+    def generate_regime_securite_sociale_parameter(condition: dict) -> ParameterNode:
+        def create_regime_dict(constraint):
+            return {
+                constraint: {
+                    'values': {
+                        date: {'value': condition[constraint]}}
+                    }}
+
+        if 'includes' in condition.keys() and 'excludes' in condition.keys():
+            raise NotImplementedError(
+                'Condition "regime_securite_sociale" does not support having'
+                ' both "includes" and "excludes" properties.\n'
+                'Please check the YAML file.'
+                )
+
+        constraint: str = 'includes' if 'includes' in condition.keys() else 'excludes'
+
+        return {condition_type: create_regime_dict(constraint)}
+
+    def generate_simple_parameter(condition: dict) -> Parameter:
+        if len(condition) == 1:
+            value = {'value': True}
+        else:
+            value = {'value': condition['values']}
+
+        return {
+            condition_type: {
+                'values': {date: value}
+                }
+            }
+
+    condition_table: dict = {
+        'age': generate_age_parameter,
+        'quotient_familial': generate_quotient_familial_parameter,
+        'regime_securite_sociale': generate_regime_securite_sociale_parameter,
+        }
+
+    date = '2020-01-01'
+
+    comparison_operators = {
+        '<=': 'maximum',
+        '>=': 'minimum',
+        '<': 'strictement_inferieur',
+        '>': 'strictement_superieur',
+        }
+
+    condition_type: str = condition['type']
+
+    if condition_type in condition_table.keys():
+        condition_parameter = condition_table[condition_type](condition)
+    else:
+        condition_parameter = generate_simple_parameter(condition)
+
+    return condition_parameter
+
+
+def conditions_to_node_data(conditions: 'list[dict]') -> dict:
+
+    conditions_formated: 'list[dict]' = [
+        condition_to_parameter(condition)
+        for condition in conditions
+        ]
+
+    data: dict = {'conditions': {}}
+
+    for condition in conditions_formated:
+        for key in condition.keys():
```

Comment:
Ca marche aussi sans le `.keys()`
```suggestion
        for key in condition:
```

Reply:
Ah oui, merci beaucoup!